plugins/affiliation/src/main.rs

```diff
@@ -0,0 +1,573 @@
+			committed_on: Ok("2024-06-20T21:00:00Z".to_string()),
+		};
+
+		let commit_3 = Commit {
+			hash: "ghi-789".to_string(),
+			written_on: Ok("2024-06-21T20:00:00Z".to_string()),
+			committed_on: Ok("2024-06-21T21:00:00Z".to_string()),
+		};
+
+		let contributor_1 = Contributor {
+			name: "John Smith".to_string(),
+			email: "jsmith@mitre.org".to_string(),
+		};
+
+		let contributor_2 = Contributor {
+			name: "Jane Doe".to_string(),
+			email: "jdoe@gmail.com".to_string(),
+		};
+
+		let commit_1_view = CommitContributorView {
+			commit: commit_1.clone(),
+			author: contributor_1.clone(),
+			committer: contributor_1.clone(),
+		};
+
+		let commit_2_view = CommitContributorView {
+			commit: commit_2.clone(),
+			author: contributor_2.clone(),
+			committer: contributor_1.clone(),
+		};
+
+		let commit_3_view = CommitContributorView {
+			commit: commit_3.clone(),
+			author: contributor_2.clone(),
+			committer: contributor_2.clone(),
+		};
+
+		let contributor_1_view = ContributorView {
+			contributor: contributor_1.clone(),
+			commits: vec![commit_1.clone(), commit_2.clone()],
+		};
+
+		let contributor_2_view = ContributorView {
+			contributor: contributor_2.clone(),
+			commits: vec![commit_2.clone(), commit_3.clone()],
+		};
+
+		let commit_1_repo = DetailedGitRepo {
+			local: repo.clone(),
+			details: Some("abc-123".to_string()),
+		};
+
+		let commit_2_repo = DetailedGitRepo {
+			local: repo.clone(),
+			details: Some("def-456".to_string()),
+		};
+
+		let commit_3_repo = DetailedGitRepo {
+			local: repo.clone(),
+			details: Some("ghi-789".to_string()),
+		};
+
+		let contributor_1_repo = DetailedGitRepo {
+			local: repo.clone(),
+			details: Some("jsmith@mitre.org".to_string()),
+		};
+
+		let contributor_2_repo = DetailedGitRepo {
+			local: repo.clone(),
+			details: Some("jdoe@gmail.com".to_string()),
+		};
+
+		let mut mock_responses = MockResponses::new();
+
+		mock_responses
+			.insert(
+				"mitre/git/commits",
+				repo,
+				Ok(vec![commit_1, commit_2, commit_3]),
+			)
+			.unwrap();
+		mock_responses
+			.insert(
+				"mitre/git/contributors_for_commit",
+				commit_1_repo,
+				Ok(commit_1_view),
+			)
+			.unwrap();
+		mock_responses
+			.insert(
+				"mitre/git/contributors_for_commit",
+				commit_2_repo,
+				Ok(commit_2_view),
+			)
+			.unwrap();
+		mock_responses
+			.insert(
+				"mitre/git/contributors_for_commit",
+				commit_3_repo,
+				Ok(commit_3_view),
+			)
+			.unwrap();
+		mock_responses
+			.insert(
+				"mitre/git/commits_for_contributor",
+				contributor_1_repo,
+				Ok(contributor_1_view),
+			)
+			.unwrap();
+		mock_responses
+			.insert(
+				"mitre/git/commits_for_contributor",
+				contributor_2_repo,
+				Ok(contributor_2_view),
+			)
+			.unwrap();
+
+		Ok(mock_responses)
+	}
+
+	#[test(tokio::test)]
+	async fn test_affiliation() {
+		let conf = Config {
+			orgs_file: pathbuf![&env::current_dir().unwrap(), "src", "example_orgs.kdl"],
```

Comment:
Should make a separate dir `test` on the same level as `src` and move the example orgs KDL there